### PR TITLE
Doc improvements

### PR DESCRIPTION
**Pull request by gonzus**

### CODE REVIEW

File: lib/Pakket.pm [hunk 1]
```diff
@@ -124,6 +126,8 @@ create them yourself or you can use the
 L<Pakket::CLI::Command::scaffold|scaffold> command to create them
 for you.
 
+NOTE: following paragraph is unclear.
```
Comment (xsawyerx) [inline lib/Pakket.pm]: You're committing a line of doc saying a paragraph in the doc is unclear? :)

**Comment by gonzus** [comment]
Yes, because I didn't know what to replace it with, but I didn't want to 
forget about it either... Fail fast.

**Comment by xsawyerx** [comment]
"Unclear" is an unclear description.

**Comment by xsawyerx** [comment]
Fixed (I hope) and merged.

**Comment by gonzus** [comment]
un·clear
ˌənˈklir/
/adjective/

 1.
    not easy to see, hear, or understand.
    "the motive for this killing is unclear"
      *
        not obvious or definite; ambiguous.
        "their future remains unclear"
        synonyms: 	uncertain
        <https://www.google.com/search?espv=2&biw=1280&bih=642&q=define+uncertain&sa=X&ved=0ahUKEwjclpWS-onQAhVGC8AKHXf6Aa8Q_SoIHjAA>,unsure
        <https://www.google.com/search?espv=2&biw=1280&bih=642&q=define+unsure&sa=X&ved=0ahUKEwjclpWS-onQAhVGC8AKHXf6Aa8Q_SoIHzAA>,unsettled
        <https://www.google.com/search?espv=2&biw=1280&bih=642&q=define+unsettled&sa=X&ved=0ahUKEwjclpWS-onQAhVGC8AKHXf6Aa8Q_SoIIDAA>,up
        in the air,debatable
        <https://www.google.com/search?espv=2&biw=1280&bih=642&q=define+debatable&sa=X&ved=0ahUKEwjclpWS-onQAhVGC8AKHXf6Aa8Q_SoIITAA>,open
        to question,in doubt,doubtful
        <https://www.google.com/search?espv=2&biw=1280&bih=642&q=define+doubtful&sa=X&ved=0ahUKEwjclpWS-onQAhVGC8AKHXf6Aa8Q_SoIIjAA>;More



        	

      *
        having or feeling doubt or confusion.
        "users are still unclear about what middleware does"
      *

**Pull request merged**